projects/objects/balls/protos/FifaSoccerBall.proto

```diff
@@ -6,10 +6,10 @@
 # tags: static
 
 PROTO FifaSoccerBall [
-  field SFVec3f    translation 0 0 0
-  field SFRotation rotation    0 1 0 0
-  field SFString   name        "fifa soccer ball"
-  field SFInt32    size        5 # Official FIFA ball sizes ranging from 1 to 5.
+  field SFVec3f            translation 0 0 0
+  field SFRotation         rotation    0 1 0 0
+  field SFString           name        "fifa soccer ball"
+  field SFInt32{1,2,3,4,5} size        5 # Defines the ball size according to FIFA standards.
 ]
 {
   %{
```

Comment:
Thank you.
You can remove this lua part as with this you are sure the value is in the correct range:
```
    if size < 1 then
      size = 1
      io.stderr:write("'size' must be strictly positive\n")
    elseif size > 5 then
      size = 5
      io.stderr:write("'size' must be less than or equal to 5\n")
    end
```